README.rst

```diff
@@ -267,6 +268,8 @@ Christopher Allen       `@bitemyapp <https://github.com/bitemyapp>`_          20
 Committee process
 -----------------
 
+The `committee bylaws <https://github.com/ghc-proposals/ghc-proposals/blob/master/committee.rst>`_ are posted.
```

Comment:
What does this sentence mean?  Maybe "Here are the committee bylaws"?   

But in any case that seems irrelevant to the process of dealing with a proposal

Reply:
Agreed. I've dropped this line; the bylaws are linked elsewhere.